src/core/server/plugins/plugins_service.mock.ts

```diff
@@ -31,7 +31,7 @@ const createSetupContractMock = (): PluginsServiceSetup => ({
 });
 const createStartContractMock = () => ({ contracts: new Map() });
 const createServiceMock = (): PluginsServiceMock => ({
-  discover: jest.fn(),
+  discover: jest.fn().mockResolvedValue([]),
```

Comment:
Shouldn't that returns a map?

https://github.com/elastic/kibana/blob/4e8ab56497cb7bc4d0fb99a72ef69f8de36c0d59/src/core/server/plugins/plugins_service.ts#L96-L97

https://github.com/elastic/kibana/blob/a8c2d5cd58f9d34dc1c33999cf1590237b25356c/src/core/server/plugins/plugins_system.ts#L48

Reply:
Yeah, replacing it with an empty map should be safer.